frontend/packages/shared/src/queryInvalidator/SyncSuccessQueriesInvalidator.ts

```diff
@@ -64,10 +66,15 @@ export class SyncSuccessQueriesInvalidator extends Queue {
     });
   }
 
-  private getCacheKeyByFileName(fileName: string): string[] {
+  private getCacheKeyByFileName(fileName: string, layoutSetName: string): string[] {
     const cacheKey = this.fileNameCacheKeyMap[fileName];
```

Comment:
Ref. previous comment, we can than do the lookup like this. :)
```suggestion
  private getCacheKeyByFileName(fileName: string): string[] {
    const cacheKey = this.fileNameCacheKeyMap[fileName] || this.folderNameCacheKeyMap[fileName];
```